integration_test/api_test/full_node_interface_test.dart

```diff
@@ -123,10 +124,10 @@ Future<void> main() async {
   final testCatCoins = [
     CoinPrototype(
       parentCoinInfo: Puzzlehash.fromHex(
-        '0fe40b1ec35f3472c8cf0f244c207c26e7a8678413dceb87cff38dc2c1c95093',
+        '98115571bab29d11253fdced97770c836ffa26fb7a8e2c98ea0f3ae1ef4282a4',
       ),
       puzzlehash: Puzzlehash.fromHex(
-        '5db372b6e7577013035b4ee3fced2a7466d6ff1d3716b182afe520d83ee3427a',
+        'ed927230abae9189ed736dfdff395a050493ba1502bd2636b8d0e1d0651e18eb',
```

Comment:
Should replace all hardcoded hex puzzle hash values with their corresponding variable values. In this case it would be in the variable `puzzlehash`